config/routes.rb

```diff
@@ -660,6 +660,7 @@ def route_actions_hash
                           controller: "projects/locations"
     resources :members, only: [:new, :create, :edit, :update, :index],
                         controller: "projects/members", param: :candidate
+    resources :project_aliases, controller: "projects/project_aliases"
```

Comment:
I think this should be `resources :aliases, controller "projects/aliases"` according to what i'm proposing above. Note that it **_doesn't_** mean that you have to namespace of the model(!) - this threw me for a long time.

Reply:
Remember when i spent probably 3 months trying to get `module Name class Description` to work? 2019 i think. Sheesh. What a weird obsession... Good times.